Sources/Rules/PrivateStateVariables.swift

```diff
@@ -26,8 +26,8 @@ public extension FormatRule {
             }
 
             // Check for @Previewable - we won't modify @Previewable macros.
-            let lineStart = formatter.startOfLine(at: stateIndex)
-            guard !formatter.tokens[lineStart ..< stateIndex].contains(where: { $0 == .keyword("@Previewable") }) else {
+            let previousToken = formatter.lastToken(before: stateIndex, where: { !$0.isSpaceOrCommentOrLinebreak })
+            guard previousToken != .keyword("@Previewable") else {
```

Comment:
I think this could be simplified further to:

```suggestion
            guard !formatter.modifiersForDeclaration(at: endOfScope, contains: "@Previewable") else {
```

I would also rename `endOfScope` (which in this case is the `let` or `var` keyword, not an end of scope token) to `keywordIndex`,

so:

```suggestion
            guard !formatter.modifiersForDeclaration(at: keywordIndex, contains: "@Previewable") else {
```

Reply:
done!